testsuite/tests/typing-extensions/open_types.ocaml.reference

```diff
@@ -87,8 +87,13 @@ Error: Signature mismatch:
          type 'a foo = ..
        Their variances do not agree.
 type exn2 = exn = ..
-Line 6, characters 8-26:
-6 | let f = function Foo -> ()
+Line 5, characters 14-20:
+5 | let _ = raise Foobar
+                  ^^^^^^
+Error: Cannot use private constructor Foobar to create values of the extensible variant type 
+       exn
+Line 7, characters 8-26:
+7 | let f = function Foo -> ()
```

Comment:
I'm sorry to ask for more work here, but would you (optionally) consider converting this test to an expect-test? The result is massively more readable, and we've been trying to slowly convert typing tests to expect-test by converting files we touch with future changes.

Reply:
No problem, is there any doc about how to do that properly ?

Reply:
Not really, but here is what you need to do:
- change "toplevel" for "expect" at the top of testsuite/tests/typing-extensions/open_types.ml
- after every `;;` in the ml file, add `[%%expect{| |}];;`
- run/promote the tests
- git rm testsuite/tests/typing-extensions/open_types.ocaml.reference

Reply:
Done.